tracing/decorators.py

```diff
@@ -42,7 +42,7 @@ def impl(test_function: Callable[[], None]):
     return impl
 
 
-def entrypoint(proj_root: pathlib.Path | None = None):
+def entrypoint(proj_root: Union[pathlib.Path, None] = None):
```

Comment:
Same goes for here.

Reply:
(Same as above) python3.9. does not allow using | as a type hint.